src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerText;
+
+  if (person.isMarried) {
+    if (person.sex === 'f') {
+      partnerText = `${person.partnerName} is my husband`;
+    } else {
+      partnerText = `${person.partnerName} is my wife`;
+    }
+  } else {
+    partnerText = 'I am not married';
+  }
```

Comment:
Looks too complicated, it's better to create a separate variable for a person's partner and calculate it based on `person.sex` field

`const partner = person.sex === 'f' ? 'husband' : 'wife`